src/ansys/tools/visualization_interface/backends/pyvista/pyvista.py

```diff
@@ -103,7 +107,11 @@ def __init__(
             use_trame = ansys.tools.visualization_interface.USE_TRAME
 
         self._use_trame = use_trame
-        self._picking_mode = picking_mode
+        self._allow_picking = allow_picking
```

Comment:
This prioritization is fine, but the user should be warned if his choices were changed. If both picking and hovering were set to True, there should some kind of warning saying that they are incompatible with each other and hovering was disabled.

Reply:
Never mind! I just saw your commit.